integration_tests/testsuite/driver.py

```diff
@@ -84,9 +84,9 @@ def _main():
             sys.exit(1)
 
         logging.debug('Deploying app!')
-        version, deploy_url = deploy_app.deploy_app(args.image, args.directory)
+        version = deploy_app.deploy_app(args.image, args.directory)
 
-    application_url = args.url or deploy_url
+    application_url = args.url or test_util.retrieve_url_for_version(version)
```

Comment:
hmm yeah, I suppose they are pretty much tied together. wouldn't really make a logical difference and reduces the number of flags to pass so I'm for it. fixed